tests/contrib/psycopg/test_psycopg.py

```diff
@@ -82,6 +85,29 @@ def assert_conn_is_traced(self, tracer, db, service):
         eq_(span.meta["out.port"], TEST_PORT)
         eq_(span.span_type, "sql")
 
+    def test_cursor_ctx_manager(self):
+        # ensure cursors work with context managers
+        # https://github.com/DataDog/dd-trace-py/issues/228
+
+        # context managers aren't supported in earlier versions of pyscopg so
+        # skip this test. should we just drop support for < 2.5?
```

Comment:
We're currently supporting `psycopg2	>= 2.4`... we should double check if someone is using that version. For now we can keep it.